docs/configuration/http_filters/router_filter.rst

```diff
@@ -50,7 +50,7 @@ x-envoy-max-retries
 
 If a retry policy is in place, setting this header controls the number of retries that Envoy will
 perform on the request's behalf. If the header is not specified and there is no :ref:`route
-configuration <config_http_conn_man_route_table_route_retry>` Envoy will perform a single retry. A
+configuration <config_http_conn_man_route_table_route_retry>` Envoy will NOT attempt any retries. A
```

Comment:
Sure. So again to clarify...

If there is no route config retry policy set and when there is no x-envoy-max-retries header, then there will be no retries.

If there is no route config retry policy set and there is no x-envoy-max-retries header BUT there is a x-envoy-retry-on header, then this will implicitly set the # of retries to one. 

is that correct?